src/types/Calculator.ts

```diff
@@ -42,6 +42,8 @@ export interface CalculationOptions {
   useElmJsonsCaching?: boolean;
   /** if true, clears ElmJsons cache before running calculation */
   clearElmJsonsCache?: boolean;
+  /** Root lib ref, to be used in calculateLibraryDataRequirements */
```

Comment:
It might be more informative if we change "root lib ref" to "reference to the root library" (or something similar)

Reply:
We should also note that it _should_ be a canonical URL, but a resource ID will also work if a matching one exists in a bundle.